### PR TITLE
styles(InputDay): migrate InputDate to vanilla css

### PR DESCRIPTION
**Pull request by guilhermespopolin**
## Describe your changes

* Migrate `InputDay` styles to use `vanilla-extract/css`

## Justify why they are needed

Gonna need it for adjusting some stuff for Accident Cross Sell.

### CODE REVIEW

File: apps/store/src/components/InputDay/InputDay.css.ts [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+  position: 'absolute',
+  left: theme.space.md,
+  top: theme.space.sm,
+  fontSize: theme.fontSizes.xs,
+  overflow: 'visible',
+  whiteSpace: 'nowrap',
+  textOverflow: 'ellipsis',
+  color: theme.colors.textSecondary,
+  selectors: {
+    '&&[data-disabled=true]': {
```
Comment (alebedev) [inline apps/store/src/components/InputDay/InputDay.css.ts:37]: What does `&&` do vs just `&`?

File: apps/store/src/components/InputDay/InputDay.css.ts [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+  paddingRight: '1.25rem',
+  paddingBottom: theme.space.sm,
+  paddingLeft: theme.space.md,
+})
+
+const chevronIconBase = style({
+  pointerEvents: 'none',
+})
+export const chevronIcon = styleVariants({
+  animated: [
```
Comment (alebedev) [inline apps/store/src/components/InputDay/InputDay.css.ts:59]: If you do class composition with clsx in React, you can remove `chevronIconBase` from every variant - this'll be less code overall

File: apps/store/src/components/InputDay/InputDay.tsx [hunk 3]
```diff
@@ -104,146 +117,40 @@ export const InputDay = (props: Props) => {
...
+          alignOffset={-4}
+        >
           {/* Fix issue on iOS where month dropdown is focused and can't be tapped */}
           <div tabIndex={1} aria-hidden={true} />
 
-          <StyledDayPicker
+          <DayPicker
             {...dayPickerProps}
+            className={dayPicker}
+            style={
```
Comment (alebedev) [inline apps/store/src/components/InputDay/InputDay.tsx:136]: I think you can assign vars in vanilla-extract styles.  It's probably better when values are static

**Review by alebedev — APPROVED** [review]
Great work!

File: apps/store/src/components/InputDay/InputDay.tsx (hunk 3 above)
Comment (guilhermespopolin) [reply apps/store/src/components/InputDay/InputDay.tsx:136]: I couldn't make it work 😢 
I mean, couldn't find a way to redefine those third party CSS variables inside `vanilla-extract`

File: apps/store/src/components/InputDay/InputDay.css.ts (hunk 2 above)
Comment (guilhermespopolin) [reply apps/store/src/components/InputDay/InputDay.css.ts:59]: That's true. Gonna update it.

File: apps/store/src/components/InputDay/InputDay.css.ts (hunk 1 above)
Comment (guilhermespopolin) [reply apps/store/src/components/InputDay/InputDay.css.ts:37]: To be honest I've just migrated that one. It seems '&&' is used to increase specificity 🤔 
We don't actually need it here so I'm going to replaced with single '&'

**Comment by guilhermespopolin** [comment]
### Merge activity

* **Apr 11, 4:05 AM EDT**: @guilhermespopolin started a stack merge that includes this pull request via [Graphite](https://app.graphite.dev/github/pr/HedvigInsurance/racoon/4144).

**Pull request merged**